classes/merlin.py

```diff
@@ -270,17 +125,36 @@ def compile(self):
             ]{}
         """.format(merlin_predicate, merlin_mbs, merlin_rates)
         return merlin_intent
- 
- 
 
- 
- 
-# Driver code
-'''* R = Ryu()
-R.compile()
- 
-K = Onos()
-K.compile()
- 
-J = Merlin()
-J.compile()'''
\ No newline at end of file
+    # overriding abstract method
+    def handle_request(self, request):
+        """ handles requests """
+        status = {
+            'code': 200,
+            'details': 'Deployment success.'
+        }
+
+        intent = request.get('intent')
+        policy = None
+        try:
+            policy, elapsed_time = self.compile(intent)
+            merlin_deployer.deploy(policy)
+        except ValueError as err:
+            print('Error: {}'.format(err))
```

Comment:
Better to use `logging` instead of prints. 

```suggestion 
            import logging
            logging.error(f'Error: {err}')
```